spec/unit/api_client_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'spec_helper'
+
+describe Typekitter::ApiClient do
+  describe '#kits_index' do
+    context 'request to typekit api is successful' do
+      it 'returns a parsed string of typekit kits' do
+        json_response = "{\"foo\":[\"success\"]}"
+        parsed_kits = JSON.parse(json_response)
+        stub_request(:get, "https://typekit.com/api/v1/json/kits").
+         with(:headers => {
+          'User-Agent'=>'Typhoeus - https://github.com/typhoeus/typhoeus',
+          'X-Typekit-Token'=>/.*/}).
+         to_return(:status => 200, :body => json_response, :headers => {})
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Use 2 (not 1) spaces for indenting an expression spanning multiple lines.